### PR TITLE
Preferences Slots for Nuke Ops, Wizards (and Admins)

### PR DESCRIPTION
**Pull request by DeityLink**
Strike Teams will also benefit from this system once I revamp them.
Alright, here's how it works:
- Open your preferences and Load an unused Save Slot
- Let's say you want your Wizards to always be called "Hocus Pocus", you'll name your slot "Wizard Hocus Pocus"
- The game will locate the "Wizard" prefix and load your preferences automatically. If you have multiple slots with the Wizard prefix, you'll be asked to choose between them.

There are 3 prefixes currently working: <b>NUK</b>, for nuke ops, <b>WIZ</b>, for wizards, and <b>ADM</b>, for admins spawning themselves as Test Dummy. Remember to insert a space after the prefix.

### CODE REVIEW

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -152,6 +152,12 @@
 	flags_inv = HIDESHOES|HIDEJUMPSUIT
 	siemens_coefficient = 2.0
 
+/obj/item/clothing/suit/chickensuit/white
+	name = "White Chicken Suit"
+	desc = "Silky Smooth. The feathers of legend...could it be..."
+	icon_state = "chickensuit_white"
+	item_state = "chickensuit_white"
```
Comment (Iamgoofball) [inline code/modules/clothing/suits/miscellaneous.dm]: :+1:

**Comment by SkowronX** [comment]
Roundstart borg, clown and mime options when

**Comment by ririchiyo** [comment]
I don't really agree with this PR at all, but that's for @D3athrow to decide what to do.

**Comment by ComicIronic** [comment]
I like the end result, but the functionality is a bit iffy, especially since you need two copies of a profile if you want it for two roles. Wouldn't it be more sensible to have a toggle on profiles that says "I would like this to be a choice for when I'm X role?"

**Comment by DeityLink** [comment]
>Wouldn't it be more sensible to have a toggle on profiles that says "I would like this to be a choice for when I'm X role?"
@ComicIronic That's actually much more complicated to set up

**Comment by ZomgPonies** [comment]
I like the IDEA of that functionality, but this seems an ass-backwards way of doing things.

**Comment by DeityLink** [comment]
For the sake of simplicity and name lengths I've reduced the prefixs to 3-letters all caps words. See OP. Might rework that later but I'm too tired right now.

**Comment by ririchiyo** [comment]
I tagged this because I'm just going to incorporate this into some of the sqlite database changes at some point and this was such a shitty way to handle this.

**Pull request closed without merging**